src/android/jni/jni_usercb.h

```diff
@@ -0,0 +1,27 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2019 Intel Corporation. All Rights Reserved.
+
+#ifndef LIBREALSENSE_JNI_USER_H
+#define LIBREALSENSE_JNI_USER_H
+
+#include <jni.h>
+#include "../../../include/librealsense2/rs.h"
+#include "../../../include/librealsense2/hpp/rs_frame.hpp"
+
+// Java user frame callback data
+struct rs_jni_cbdata
```

Comment:
Please consider frame_callback_data instead of rs_jni_cbdata (or some mix of these if you want to).